README.md

```diff
@@ -30,4 +30,4 @@ See the [Contributing guide](CONTRIBUTING.md) for how to contribute to this proj
 
 Development documentation can be found in [Cardano Node Wiki](https://github.com/input-output-hk/cardano-node-wiki/wiki).
 
-Haddock documentation is available at: https://cardano-cli.cardano.intersectmbo.org/
+Haddock documentation is available at: https://intersectmbo.github.io/cardano-cli/
```

Comment:
The correct url now is: https://cardano-api.cardano.intersectmbo.org/

Reply:
Ah I see, this link was broken initially, but now it works. So this removes a change from this PR.